searchcorespi/src/tests/index/active_disk_indexes/active_disk_indexes_test.cpp

```diff
@@ -48,6 +70,90 @@ TEST_F(ActiveDiskIndexesTest, is_active_returns_false_for_bad_name)
     EXPECT_FALSE(isActive("index.flush.0"));
 }
 
+TEST_F(ActiveDiskIndexesTest, remove_works)
+{
+    EXPECT_TRUE(remove(IndexDiskDir()));
+    auto fusion1 = get_index_disk_dir("index.fusion.1");
+    EXPECT_TRUE(remove(fusion1));
+    add_not_active(fusion1);
+    EXPECT_TRUE(remove(fusion1));
+    setActive("index.fusion.1", 0);
+    EXPECT_FALSE(remove(fusion1));
+    notActive("index.fusion.1");
+    EXPECT_TRUE(remove(fusion1));
+}
+
+TEST_F(ActiveDiskIndexesTest, basic_get_transient_size_works)
+{
+    setActive("index.fusion.1", 1000000);
```

Comment:
Please add comment describing which index collection setup this represents.

Reply:
Done.